### PR TITLE
Respect FLY_ORG env var

### PR DESCRIPTION
**Pull request by redjonzaci**
### Change Summary

What and Why:
Try to get the orgName from FLY_ORG env var first.

How:

Related to: #2757

---

### Documentation

- [ ] Fresh Produce
- [ ] In superfly/docs, or asked for help from docs team
- [ ] n/a

### CODE REVIEW

**Comment by redjonzaci** [comment]
If a wrong org is provided to FLY_ORG, we get `Error: error getting organization: Could not find`.

**Comment by redjonzaci** [comment]
Now if FLY_ORG is not provided, it moves on like before.
I wonder if there's another case that need to be covered.
If not, maybe the code could be improved further.

**Review by dangra — CHANGES_REQUESTED** [review]
I'd standardize for `flag.GetOrg(ctx)` and bake `env.First("FLY_ORG")` in there. 

So, any future use of `--org` flag will inherit the correct behavior as long as it uses `flag.GetOrg` to fetch its value.

**Comment by redjonzaci** [comment]
Thanks @dangra! Getting to this soon.

File: internal/flag/context.go [hunk 1]
```diff
@@ -118,6 +119,10 @@ func IsSpecified(ctx context.Context, name string) bool {
 
 // GetOrg is shorthand for GetString(ctx, Org).
 func GetOrg(ctx context.Context) string {
+	org := env.First("FLY_ORG")
+	if org != "" {
+		return org
+	}
 	return GetString(ctx, flagnames.Org)
```
[the remark this answers is not in the document]
Comment (redjonzaci) [reply internal/flag/context.go:126]: Sure

File: internal/command/orgs/orgs.go [hunk 2]
```diff
@@ -94,9 +94,13 @@ func slugFromArgOrSelect(ctx context.Context, orgSlug string, filters ...api.Org
 }
 
 func OrgFromFirstArgOrSelect(ctx context.Context, filters ...api.OrganizationFilter) (*api.Organization, error) {
```
Comment (redjonzaci) [inline internal/command/orgs/orgs.go:96]:
```suggestion
func OrgFromFirstArgOrSelect(ctx context.Context, filters ...api.OrganizationFilter) (*api.Organization, error) {
```
I feel like after my changes this function name is not honest anymore.

Comment (redjonzaci) [reply internal/command/orgs/orgs.go:96]: Thought to rename it to `OrgFromEnvVarOrFirstArgOrSelect`

**Review by dangra — APPROVED** [review]
well done!

**Comment by michaeldwan** [comment]
Thanks for the PR!

**Pull request merged**